### PR TITLE
IDESSCD updates

### PR DESCRIPTION
**Pull request by steininger**
add TotalTarFileSize to export.
add description.
clarify async function-names.

### CODE REVIEW

File: src/fiskaltrust.ifPOS/v1/de/Models/StartExportSessionByTransactionRequest.cs [hunk 1]
```diff
@@ -6,9 +6,12 @@ namespace fiskaltrust.ifPOS.v1.de
     public class StartExportSessionByTransactionRequest
     {
         [DataMember(Order = 10)]
-        public ulong From { get; set; }
+        public string ClientId { get; set; } = string.Empty;
```
Comment (TSchmiedlechner) [inline src/fiskaltrust.ifPOS/v1/de/Models/StartExportSessionByTransactionRequest.cs:9]: What's the reason for using this default value here?

Comment (steininger) [reply src/fiskaltrust.ifPOS/v1/de/Models/StartExportSessionByTransactionRequest.cs:9]: no reason, i will remove it.
as trials shown, null works fine for our major supported tse-devices.
thx.

**Pull request merged**